tests/integration/device.js

```diff
@@ -0,0 +1,26 @@
+
+const viewports = {
+  mobile: {
```

Comment:
Maybe we should set an explicit viewport size for desktop too ?

Reply:
Yep, better be explicit. I'll add it.